File: src/question/month08/day0806/ms_0806.cpp
#include <iostream>

using namespace std;

int main() {
	int data[10], min = 1000, i = 0;

	while (i < 10) {
		scanf_s("%d", &data[i]);

		if (data[i] > 1000)
			continue;

		i++;
	}

	for (int i = 0; i < 10; i++) {
		if (data[i] < min)
			min = data[i];
	}

	cout << min << endl;

	return 0;
}

File: year2020/month08/day0806/ms_0806.cs
using System;

namespace algorithm_29
{
    class ms_0806
    {
        static void Main(string[] args)
        {
            int[] data = new int[10];

            for(int i = 0; i < data.Length; i++) {
                int temp = Convert.ToInt32(Console.ReadLine());
                
                if(temp > 1000) {
                    i--;
                    continue;
                }

                data[i] = temp;
            }

            int min = 1000;

            for(int i = 0; i < data.Length; i++) {
                if(data[i] < min) {
                    min = data[i];
                }
            }

            Console.WriteLine(min);
        }
    }
}
